### PR TITLE
generate bundle and catalog, add release command

### PR DESCRIPTION
**Pull request by jgbernalp**
This PR creates the missing bundle dockerfile to build a bundle. It also adds a release command that builds and deploys the bundle and catalog

### CODE REVIEW

**Comment by Nexucis** [comment]
Otherall it looks fine, I just don't understand what we have to publish.(certainly because I am not familiar with the release and deployment workflow of the operator)

Do we need to publish also a docker image containing the operator binary (a.k.a build thanks to `main.go`) ?
Or actually it's not necessary and we just need to publish the `bundle` image ?
Or do we need to publish both ?

Asking that because if we only need to publish the `bundle` image, I can image we don't need goreleaser anymore to be able to build docker manifest for arm64 and amd64

**Review by Nexucis — APPROVED** [review]
cool thanks for the explanations @jgbernalp it's clearer now !

File: config/manifests/bases/perses-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.arm64: supported
+    operatorframework.io/arch.ppc64le: supported
```
[the remark this answers is not in the document]
Comment (jgbernalp) [reply config/manifests/bases/perses-operator.clusterserviceversion.yaml:10]: ok will adjust.

**Review by Nexucis — APPROVED** [review]
👍

**Pull request merged**